blackbox/requirements.txt

```diff
@@ -8,19 +8,19 @@ asyncpg>=0.20.1
 cr8>=0.19.1
 tqdm==4.24.0
 sphinx-csv-filter>=0.2.0
-pycodestyle==2.4.0
+pycodestyle==2.6.0
 zc.customdoctests==1.0.1
 minio>=5.0.0
 
 # packages for local dev
 
-sphinx-autobuild==0.7.1
+sphinx-autobuild==2020.9.1
 
 # the next section should mirror the RTD environment
 
 alabaster>=0.7,<0.8,!=0.7.5
 setuptools<41
-sphinx==1.8.5
+sphinx==3.3.1
```

Comment:
I don't know whether this will break something on the downstream docs infrastructure or if it is okay to just bump within this repository. Please let me know.

Reply:
FWIW, I don't see any brokenness on https://crate--10831.org.readthedocs.build/en/10831/, which apparently has been derived from this patch.